tests/atomizer.js

```diff
@@ -1071,6 +1071,34 @@ describe('Atomizer()', function () {
             const result = atomizer.getCss(config);
             expect(result).to.equal(expected);
         });
+        it ('has basic suppport for css variables', function () {
+            const atomizer = new Atomizer();
+            const config = {
+                breakPoints: {
+                    sm: '@media screen and (min-width:700px)'
+                },
+                classNames: [
+                    'C(--brand-color)',
```

Comment:
Could also add test cases for identifiers with no hyphens, or other valid chars

Reply:
https://drafts.csswg.org/css-values-4/#identifier-value

Reply:
Added more tests in https://github.com/acss-io/atomizer/pull/395/commits/5dc7cbc932bc7aebbcecef421bf95ea453a911eb